### PR TITLE
pfnCI (pytest)

### PR DESCRIPTION
**Pull request by Hakuyume**

### CODE REVIEW

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by 69guitar1015** [comment]
ChainerCI, test this please

**Comment by yuyu2172** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by yuyu2172** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
ChainerCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Comment by Hakuyume** [comment]
pfnCI, test this please

File: chainercv/utils/testing/parameterized.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import six
+
+from chainer import testing
+
+
+def parameterize(*params):
```
Comment (yuyu2172) [inline chainercv/utils/testing/parameterized.py]: Can you add comment on why this method is needed?

Comment (Hakuyume) [reply chainercv/utils/testing/parameterized.py]: updated

**Comment by Hakuyume** [comment]
pfnCI, test this please

**Pull request merged**